easybuild/easyconfigs/c/Commet/Commet-20150415-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp' 
+
+name = 'Commet' 
+version = '20150415' 
+commit = '4ef0705'
+versionsuffix = '-Python-%(pyver)s' 
+
+homepage = 'https://colibread.inria.fr/software/commet/' 
+description = """ COMMET ("COmpare Multiple METagenomes") provides a global similarity overview between all datasets of
+ a large metagenomic project. 
+""" 
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/pierrepeterlongo/commet/archive']
+sources = ['%s.tar.gz' % commit]
+
+checksums = ['19cba09ca7a92eaed75ed6af820d9451'] 
+
+dependencies = [
+    ('Python', '2.7.11'), 
+] 
+
+files_to_copy = ["bin","doc","include","ABCDE_bench",(["Commet.py"],'bin')] 
+
+sanity_check_paths = {
+    'files': ['bin/%s' % binfile for binfile in ['Commet.py', 'bvop', 'compare_reads',
+	                                             'extract_reads', 'filter_reads', 'index_and_search']],
```

Comment:
@RvDijk please remove the `<tab>` character here in the front  (we will start enforcing the lack of tabs soonish)